designer/client/src/actions/nk/liveData.ts

```diff
@@ -31,15 +31,15 @@ export const stopLiveData = (): ThunkAction => (dispatch) => {
     clearTimeout(refreshTimeout);
 };
 
-export function fetchAndDisplayLiveData(refreshIn: number | false = 1): ThunkAction {
+export function fetchAndDisplayLiveData(refreshIn: number | false = 1, showErrors = false): ThunkAction {
```

Comment:
Quickfix to only show error toasts when user click the LiveData button and something goes wrong. Errors are not shown when refresh is happening in the background. 

It is only a quickfix and will have to be be handled properly/